submissions/ustymchyk/js-post-oop/index.js

```diff
@@ -35,12 +39,12 @@ class Pet extends Creature {
     #angrySound;
     #energy;
 
-    constructor(name, gender, age, happySound, angrySound) {
-        if (new.target === Pet) {
+    constructor(name, gender, age, species, happySound, angrySound) {
+        if (new.target === Creature) {
```

Comment:
I can remember you fixed this to have a comparison against `Pet`. It didn't work as expected?

Reply:
No, I`ve fixed it in the 'submission' repo and forgot in my repo. Then, when made improvements in my repo, forgot about this point. So, it just a mistake during working with two repo.